core/src/main/kotlin/org/evomaster/core/problem/rest/classifier/inputEncoder.kt

```diff
@@ -0,0 +1,48 @@
+package org.evomaster.core.problem.rest.classifier
+
+import org.evomaster.core.problem.rest.data.RestCallAction
+import org.evomaster.core.search.gene.BooleanGene
+import org.evomaster.core.search.gene.collection.EnumGene
+import org.evomaster.core.search.gene.numeric.DoubleGene
+import org.evomaster.core.search.gene.numeric.IntegerGene
+
+
+// Encoding IntegerGene, DoubleGene, EnumGene, and BooleanGene of a RestCallAction into a double vector
```

Comment:
comments on functions and fields should be within a `/**  */` (note the initial double-*), at it would make them into JavaDoc, that are then picked up by tools and IDEs (eg when hovering over such elements with mouse)